Portfolio.css

```diff
@@ -1,3 +1,126 @@
+* {
+    box-sizing: border-box;
+}
+
 Body {
     font-family: 'Quicksand', sans-serif;
+    font-size: 62.5%;
+    background-color: #fff;
+    color: #5D6063;
+}
+
+.main-nav {
+    background-color: #ffffff;
+    color: #5D6063;
+    position: fixed;
+    box-shadow: 0 2px 0 rgba(93, 96, 99, .1);
+    width: 100%;
+    font-size: 1rem;
+    padding: 10px;
+}
+
+.nav-button-left {
+    float: left;
+    position: relative;
+    left: 5px;
+    bottom: 5px;
+}
+
+.nav-button-right {
+    float: right;
+    position: relative;
+    right: 25px;
+    bottom: 5px;
+}
+
+.nav-text {
+    margin-left: 15px;
+}
+
+.main-header-wrapper {
+    max-width: 580px;
+    margin-left: auto;
```

Comment:
Setting padding/margin for only top & bottom, or left & right, can be done on 1 line.